src/user/UserForm.tsx

```diff
@@ -145,13 +145,13 @@ export const UserForm = ({
             setOpen(false);
           }}
           onClose={() => setOpen(false)}
-          filterGroups={selectedGroups}
+          filterGroups={selectedGroups.map((group) => group.name) as string[]}
```

Comment:
If you are sure that `name` is always present you can use a non-null assertion so there is no need to type-cast.
```suggestion
          filterGroups={selectedGroups.map((group) => group.name!)}
```